README.md

```diff
@@ -103,24 +112,45 @@ easy_install --upgrade "ibmcloudant>=0.0.27"
+account.
 
 1. `IAM`, `COUCHDB_SESSION`, `BASIC` or `NOAUTH` **authentication type**.
-    1. [*IAM authentication*](#iam-authentication) is highly recommended when your back-end database server is [**Cloudant**](https://cloud.ibm.com/docs/Cloudant?topic=cloudant-ibm-cloud-identity-and-access-management-iam-). This authentication type requires a server-generated `apikey` instead of a user-given password.
-    1. [*Session cookie (`COUCHDB_SESSION`) authentication*](#session-cookie-authentication) is recommended for [Apache CouchDB](https://docs.couchdb.org/en/stable/api/server/authn.html#cookie-authentication) or for [Cloudant](https://cloud.ibm.com/docs/Cloudant?topic=cloudant-authentication#cookie-authentication) when IAM is unavailable. It exchanges username and password credentials for an `AuthSession` cookie from the `/_session` endpoint.
-    1. [*Basic* (or legacy) *authentication*](#basic-authentication) is a fallback for both [Cloudant](https://cloud.ibm.com/docs/services/Cloudant/api?topic=cloudant-authentication#basic-authentication) and [Apache CouchDB](https://docs.couchdb.org/en/stable/api/server/authn.html#basic-authentication) back-end database servers. This authentication type requires the good old `username` and `password` credentials.
-    1. *Noauth* authentication does not need any credentials. Note that this authentication type will only work for queries against a database with read access for everyone.
-1. The service `url`
+1. [*IAM authentication*](#iam-authentication) is highly recommended when your
+back-end database server is [**Cloudant**][cloud-IAM-mgmt]. This
+authentication type requires a server-generated `apikey` instead of a
+user-given password.
+1. [*Session cookie (`COUCHDB_SESSION`) authentication*](#session-cookie-authentication)
+is recommended for [Apache CouchDB][couch-cookie-auth] or for
+[Cloudant][cloudant-cookie-auth] when IAM is unavailable. It exchanges username
+and password credentials for an `AuthSession` cookie from the `/_session`
+endpoint.
+1. [*Basic* (or legacy) *authentication*](#basic-authentication) is a fallback
+for both [Cloudant][cloudant-basic-auth] and [Apache CouchDB][couch-basic-auth]
+back-end database servers. This authentication type requires the good old
+`username` and `password` credentials.
+1. *Noauth* authentication does not need any credentials. Note that this
+authentication type will only work for queries against a database with read
+access for everyone.
+1. The service `url`.
 
 You have to add these properties as your **environment variables**, because some
 examples that follow assume that these variables are set.
-To learn more about authentication configuration see the related documentation in the
-[Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant#authentication?code=python) or in the
-[general SDK usage information](https://github.com/IBM/ibm-cloud-sdk-common#authentication).
 
 #### IAM authentication
```

Comment:
I would highlight this is an example for the environment variables usage.

Reply:
I made separate sections for these 3 ways, so all the authentication examples with env vars will be one level down under the _Authenticate with environment variables_ section.

I added a sentence above that highlight env var auth is going to be used in the code examples.